metadata-ingestion/src/datahub/ingestion/source/gc/soft_deleted_entity_cleanup.py

```diff
@@ -19,7 +19,7 @@
 
 logger = logging.getLogger(__name__)
 
-QUERY_QUERY_ENTITY = """
+QUERY_ENTITIES = """
 query listQueries($input: ScrollAcrossEntitiesInput!) {
```

Comment:
Mostly cosmetic but we may rename this too

```suggestion
query listEntities($input: ScrollAcrossEntitiesInput!) {
```

Reply:
done